cmd/build.go

```diff
@@ -116,7 +134,9 @@ An optional flag --tag/-t is now available to override the resulting docker imag
 		}
 		// END DEPRECATION HANDLING
 
-		queueAndBuild(buildConfig, chain, org, ref, tag, latest, local, number, parallel)
+		queueAndBuild(buildConfig, chain, org, repo, repoHost,
```

Comment:
If using a config struct, you could simply pass in that single struct. 

That way, it's easier to extend in the future. And, you have less worry of mixing up arguments (esp. since these are mostly strings.)